modules/core/src/utils/pick-info.js

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2015 - 2017 Uber Technologies, Inc.
```

Comment:
Per discussion, let's move these from `utils/` to `lib/picking` as these are tightly coupled with core functionalities.